lms/djangoapps/discussion_api/tests/utils.py

```diff
@@ -18,7 +18,12 @@ def callback(request, _uri, headers):
         data along with the data from response_overrides and dummy values for any
         additional required fields.
         """
+        is_new_thread = thread_data.pop("new_thread")
         response_data = make_minimal_cs_thread(thread_data)
+        # pop resp_total as it is not part of response from CS on thread creation
+        # request
+        if is_new_thread:
+            response_data.pop("resp_total")
```

Comment:
@nasthagiri @BenjiLee 
1. Yes, comment service does not return `resp_total` while creation; otherwise it does.
2. I have updated comment service to return `resp_total` for both question and discussion type threads. This way the output will be consistent and we do not have to go through overhead of adding/removing response_count/resp_total at discussion api side.
For question type thread; `resp_total` will be sum of `non_endorsed_responses` and `endorsed_responses`. I have also tested web discussion forum to make sure they do not break when adding `resp_total` in response of get question type thread and it seems to be working fine.

Let me know if there is any concern about updating comment service.